src/main/resources/jenkins/branch/BranchSource/config.jelly

```diff
@@ -23,18 +23,18 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:branch="/lib/branch-api">
-  <tr name="source">
-    <td colspan="3">
+  <branch:rowWrapper name="source">
+    <branch:entryWrapper>
```

Comment:
Oops, this needs `colspan="3"`. Right now when running `branch-api` 2.6.1 against old versions of core it looks like this:

<img width="950" alt="Screen Shot 2020-11-09 at 16 41 04" src="https://user-images.githubusercontent.com/1068968/98599678-61787100-22aa-11eb-9805-edc9e9f0029f.png">